packages/lit-html/src/lit-html.ts

```diff
@@ -205,13 +207,11 @@ const HTML_RESULT = 1;
 const SVG_RESULT = 2;
 
 /** TemplatePart types */
-// TODO (justinfagnani): since these are exported, consider shorter names,
-// like just `ATTRIBUTE`.
-export const ATTRIBUTE_PART = 1;
-export const NODE_PART = 2;
-export const PROPERTY_PART = 3;
-export const BOOLEAN_ATTRIBUTE_PART = 4;
-export const EVENT_PART = 5;
+const ATTRIBUTE_PART = 1;
```

Comment:
Did you change your mind on putting these in an object literal?

Reply:
Oh right...

Reply:
done